flag_groups.go

```diff
@@ -201,7 +211,11 @@ func validateExclusiveFlagGroups(data map[string]map[string]bool) error {
 
 		// Sort values, so they can be tested/scripted against consistently.
 		sort.Strings(set)
-		return fmt.Errorf("if any flags in the group [%v] are set none of the others can be; %v were all set", flagList, set)
+		return &FlagGroupError{
+			flagList:      flagList,
+			flagGroupType: FlagsAreMutuallyExclusive,
+			problemFlags:  set,
+		}
```

Comment:
```suggestion
		return FlagGroupError{
			flagList:      flagList,
			flagGroupType: FlagsAreMutuallyExclusive,
			problemFlags:  set,
		}
```

Please read this

https://github.com/spf13/cobra/pull/2266#discussion_r2074792615